src/AccessibilityInsights/MainWindowHelpers/Initialize.cs

```diff
@@ -51,13 +54,13 @@ void StartStartMode()
             InitTimerAutoSnap();
             InitAccessKeyRule();
             InitHighlighter();
-
+            this.gdModes.Visibility = Visibility.Visible;
+            this.btnPause.Visibility = Visibility.Visible;
             /// Set UI appropriately if showing startup screen
             if (ConfigurationManager.GetDefaultInstance().AppConfig.NeedToShowWelcomeScreen())
             {
-                this.bdLeftNav.IsEnabled = false;
-                this.gdModes.Visibility = Visibility.Collapsed;
-                this.ctrlStartUpMode.ShowControl();
+                //this.bdLeftNav.IsEnabled = false;
+                _ = ctrlDialogContainer.ShowDialog(new StartUpModeControl()).ContinueWith(BackToSelectingOnDispatcher, TaskScheduler.Current);
```

Comment:
What's the purpose of `_ =` ?

Reply:
I needed it earlier but made other changes that rendered it obsolete.

Reply:
Removed